source/agora/node/Runner.d

```diff
@@ -29,11 +29,14 @@ import vibe.web.rest;
 
 import std.file;
 import std.format;
+import std.typecons : Tuple, tuple;
 
 import core.time;
 
 mixin AddLogger!();
 
+alias NodeListenerTuple = Tuple!(FullNode, "node", HTTPListener, "http_listener");
```

Comment:
```suggestion
///
public alias NodeListenerTuple = Tuple!(FullNode, "node", HTTPListener, "http_listener");
```

The `///` generates documentation, and we use explicit protection attributes for all (most) symbols.